modules/@apostrophecms/busy/ui/apos/components/TheAposBusy.vue

```diff
@@ -14,7 +14,7 @@ export default {
     };
   },
   mounted() {
-    apos.bus.$on('apos-busy', state => {
+    apos.bus.$on('busy', state => {
```

Comment:
Apostrophe events are specific to apostrophe by definition, we do not namespace any others like this, so I removed the unwanted prefix.